From c64bead45fcad513de895e605fff65a2e54964a6 Mon Sep 17 00:00:00 2001
From: Tim Blechmann <tim.blechmann@qt.io>
Date: Wed, 24 Sep 2025 10:20:21 +0800
Subject: [PATCH 079/239] Tests: gifs - include missing header

QSignalSpy is used, but the header is not included. This patch fixes a
compile error when building manual tests.

Pick-to: 6.8
Change-Id: I8e6fc644aa73ff3d926a61687122fa210bcf5cd9
Reviewed-by: Mitch Curtis <mitch.curtis@qt.io>
(cherry picked from commit 6cb8129bb5b3c4b028ed8256f6a45f29ea258501)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 tests/manual/quickcontrols/gifs/gifrecorder.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tests/manual/quickcontrols/gifs/gifrecorder.cpp b/tests/manual/quickcontrols/gifs/gifrecorder.cpp
index 26ccec0360..72374f8ef5 100644
--- a/tests/manual/quickcontrols/gifs/gifrecorder.cpp
+++ b/tests/manual/quickcontrols/gifs/gifrecorder.cpp
@@ -7,6 +7,7 @@
 #include <QQmlComponent>
 #include <QQuickItem>
 #include <QTest>
+#include <QtTest/qsignalspy.h>
 
 /*!
     QProcess wrapper around byzanz-record (sudo apt-get install byzanz).
-- 
2.51.2

